src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Methodologies/MethodologyController.cs

```diff
@@ -119,5 +123,13 @@ public Task<ActionResult<List<MethodologyStatusViewModel>>> GetMethodologyStatus
                 .GetMethodologyStatuses(methodologyVersionId)
                 .HandleFailuresOrOk();
         }
+        
+        [HttpGet("methodology/approvals")]
+        public async Task<ActionResult<List<MethodologyVersionViewModel>>> ListMethodologyVersionsForApproval()
+        {
+            return await _methodologyService
+                .ListMethodologyVersionsForApproval(_userService.GetUserId())
```

Comment:
Haha well see my other response to one of your other bits of feedback!  I'd be happy to refactor so that it gets grabbed from within the service method itself to prevent accidental misuse and dodging of permission checks.

Reply:
I've internalised the fetching of the User's Id into this method.